.gitignore

```diff
@@ -3,3 +3,4 @@
 /.phpunit.result.cache
 /.idea/
 /.php-cs-fixer.cache
+/.run/
```

Comment:
Where does this file come from?

Reply:
von den PHPStorm test-configuration

Reply:
Never saw it myself, let's review your config